src/posts/posts.service.ts

```diff
@@ -81,7 +119,9 @@ export class PostsService {
     return response;
   }
   async getAllPosts(): Promise<Posts> {
-    const currentPosts = await this.postRepository.getAllPosts();
+    let currentPosts = await this.postRepository.getAllPosts();
```

Comment:
Please refactor this method as per our discussion.

Reply:
done